target/zephyr/Kconfig

```diff
@@ -469,7 +469,7 @@ if MENDER_MCU_CLIENT
                 range 0 2
                 default 2
                 help
-                    Peer verification level for TLS connection.
+                    Peer verification level for TLS connection. For testing purposes, set to 0 to skip TLS peer verify
```

Comment:
So what does `1` mean?

Reply:
It means optional. See [here](https://github.com/Mbed-TLS/mbedtls/blob/1782587af27c693e6f0faf5851d76c75257bec08/include/mbedtls/ssl.h#L329) and [here](https://github.com/Mbed-TLS/mbedtls/blob/1782587af27c693e6f0faf5851d76c75257bec08/include/mbedtls/ssl.h#L2035).

Reply:
IMO we don't really support "optional", because to properly use it we would need to check the handshake result and print a warning or something.

Reply:
Right. The `help` should say it then. And perhaps create a task to make sure `1` is handled gracefully?

Reply:
> The help should say it then.

Amended

> And perhaps create a task to make sure 1 is handled gracefully?

[Done](https://northerntech.atlassian.net/browse/MEN-8413).